src/commands/cmd_cluster.cc

```diff
@@ -338,9 +338,19 @@ class CommandReadWrite : public Commander {
   }
 };
 
+class CommandAsking : public Commander {
+ public:
+  Status Execute(Server *srv, Connection *conn, std::string *output) override {
+    conn->EnableFlag(redis::Connection::kAsking);
+    *output = redis::SimpleString("OK");
```

Comment:
LGTM, it would be nice to write a simple comment that says we dare not do anything, just to be compatible.

Reply:
It should align with the Redis behavior for now, since it only allows the ASKING flag connection to query keys of the importing slot.

Reply:
Roger that.